pkg/controllers/static/deprovisioning/controller.go

```diff
@@ -41,7 +45,13 @@ import (
 	"sigs.k8s.io/karpenter/pkg/cloudprovider"
 	"sigs.k8s.io/karpenter/pkg/controllers/state"
 	"sigs.k8s.io/karpenter/pkg/operator/injection"
+	disruptionutils "sigs.k8s.io/karpenter/pkg/utils/disruption"
 	nodepoolutils "sigs.k8s.io/karpenter/pkg/utils/nodepool"
+	"sigs.k8s.io/karpenter/pkg/utils/pod"
+)
+
+const (
+	TerminationReason = "deprovisioned"
```

Comment:
I'm not in love with this termination reason name that we have given -- can we do something like `scale_down` or `overprovisioned` or something else which better fits a case where we have a static NodePool that has too many nodes against the desired replicas